app/src/main/java/co/develhope/chooseyouowncocktail_g2/ui/home/HomeFragment.kt

```diff
@@ -37,26 +31,35 @@ class HomeFragment : Fragment() {
             ViewModelProvider(this).get(HomeViewModel::class.java)
 
         _binding = FragmentHomeBinding.inflate(inflater, container, false)
-
-        val headerAdapter = HeaderAdapter()
-
+        val context = requireActivity()
+        val imOn =
+            context.resources.getIdentifier("ic_fav_off", "drawable", context.packageName)
+        val imOff =
+            context.resources.getIdentifier("ic_fav_on", "drawable", context.packageName)
```

Comment:
Absolutely not, this isn't the right method used to referencing a drawable...

You need to use something like this 
        setIconDrawable(ContextCompat.getDrawable(context, your_drawable_resource))
 
 In the courses there is an entire video dedicated to the resources, and how to referencing them.